pulp_rpm/tests/functional/api/test_consume_content.py

```diff
@@ -111,11 +132,11 @@ def test_with_xml_base_in_metadata(self):
         completely different base path when looking up relative paths.
         """
         delete_orphans()
-        self.do_test("immediate", mirror=False, url=REPO_WITH_XML_BASE_URL)
+        self.do_test("immediate", sync_policy="mirror_content_only", url=REPO_WITH_XML_BASE_URL)
```

Comment:
it probably does not matter whether the policy is additive or mirror_content_only in this testcase?

Reply:
Right, it does not.  But this is expected to fail with metadata mirroring so I figured I ought to use a mirror mode.  Perhaps it would make sense to use both.